cache/cache.go

```diff
@@ -20,6 +20,12 @@ type Interface interface {
 	Purge(key *Key) error
 }
 
+// Wrapped Cache are for cache backend that wrap other cache backends
+// Original will return the first cache backend to be wrapped
+type Wrapped interface {
```

Comment:
Maybe this question will be answered in some of the other PRs, but what's the plan for this interface?

Reply:
Got it. This could maybe be moved to the cache PR then. No biggie either way, I just didn't see it being used with this update which is why I asked about it.

Reply:
I think I will leave here. But good point. The only reason I'm not removing it, is because I have to move it across three branches and rebases, and I'm running out of time.

Reply:
That's fair.